fdbclient/BlobCipher.cpp

```diff
@@ -1910,10 +1910,9 @@ void testKeyCacheEssentials(DomainKeyMap& domainKeyMap,
 		Reference<BaseCipher> baseCipher = domainKeyMap[minDomainId][minBaseCipherKeyId];
 		uint8_t rawCipher[baseCipher->len];
 		memcpy(rawCipher, baseCipher->key.get(), baseCipher->len);
-		// modify few bytes in the cipherKey
-		for (int i = 2; i < 5; i++) {
-			rawCipher[i]++;
-		}
+		// modify cipherKey by flipping a bit
+		const int idx = deterministicRandom()->randomInt(0, baseCipher->len + 1);
```

Comment:
In this case you don't need the +1 because we aren't using the result as an exclusive upper bound

```suggestion
		const int idx = deterministicRandom()->randomInt(0, baseCipher->len);
```